Dfe.Academies.Academisation.Service/Commands/ProjectGroup/SetProjectGroupCommandHandler.cs

```diff
@@ -1,56 +1,44 @@
-﻿using Dfe.Academies.Academisation.Core.Utils;
-using Dfe.Academies.Academisation.Core;
+﻿using Dfe.Academies.Academisation.Core;
 using Dfe.Academies.Academisation.Domain.ProjectGroupsAggregate;
 using MediatR;
 using Microsoft.Extensions.Logging;
 using Dfe.Academies.Academisation.Domain.ApplicationAggregate;
-using Dfe.Academies.Academisation.Service.Extensions;
 using FluentValidation;
 
 namespace Dfe.Academies.Academisation.Service.Commands.ProjectGroup
 {
-	public class SetProjectGroupCommandHandler(IProjectGroupRepository projectGroupRepository, IDateTimeProvider dateTimeProvider, IValidator<SetProjectGroupCommand> validator, ILogger<SetProjectGroupCommandHandler> logger, IConversionProjectRepository conversionProjectRepository) : IRequestHandler<SetProjectGroupCommand, CommandResult>
+	public class SetProjectGroupCommandHandler(IProjectGroupRepository projectGroupRepository, ILogger<SetProjectGroupCommandHandler> logger, IConversionProjectRepository conversionProjectRepository) : IRequestHandler<SetProjectGroupCommand, CommandResult>
 	{
 		public async Task<CommandResult> Handle(SetProjectGroupCommand message, CancellationToken cancellationToken)
 		{
-			logger.LogError($"Setting project group with urn:{message.GroupReferenceNumber}");
-			var validationResult = validator.Validate(message);
-			if (!validationResult.IsValid)
-			{
-				logger.LogError($"Validation failed while setting project group:{message}");
-				return new CommandValidationErrorResult(validationResult.Errors.Select(r => new ValidationError(r.PropertyName, r.ErrorMessage)));
-			}
+			logger.LogInformation("Setting project group with reference number:{value}", message.GroupReferenceNumber);
+
 			var projectGroup = await projectGroupRepository.GetByReferenceNumberAsync(message.GroupReferenceNumber, cancellationToken);
 			if (projectGroup == null)
 			{
-				logger.LogError($"Project group is not found with urn:{message.GroupReferenceNumber}");
+				logger.LogError("Project group is not found with reference number:{value}", message.GroupReferenceNumber);
 				return new NotFoundCommandResult();
 			}
-			// we need to change this to be the assign user command
-			//projectGroup.SetAssignedUser(message.TrustUrn, dateTimeProvider.Now);
-
-			projectGroupRepository.Update(projectGroup);
-			await projectGroupRepository.UnitOfWork.SaveChangesAsync(cancellationToken);
 
-			var conversionProjects = await conversionProjectRepository.GetConversionProjectsByIds(message.ConversionsUrns, cancellationToken).ConfigureAwait(false);
+			var conversionProjects = await conversionProjectRepository.GetConversionProjectsByProjectGroupIdAsync(projectGroup.Id, cancellationToken).ConfigureAwait(false);
```

Comment:
@paullocknimble This function wouldn't let to update Project group with newly added conversion projects in DB through FE. Unless, FE is providing list of conversion project ids as a separate property and updating them